.gitignore

```diff
@@ -121,5 +121,7 @@ Config/secrets.xcconfig
 **/*.otf
 **/.swiftpm
 **/.tmp
+
 schema.graphqls
 introspection_response.json
+!PocketKit/Sources/PocketGraph/user-defined-operations
```

Comment:
Is there a conflicting ignore where this had to specifically be added?

Reply:
No, at one point i had ignored the whole graph folder